cmd/clairctl/clair/versions.go

```diff
@@ -0,0 +1,32 @@
+package clair
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+)
+
+func Versions() (interface{}, error) {
+	Config()
+	response, err := http.Get(uri + "/versions")
```

Comment:
This API request doesn't exist anymore. We plan to re-introduce it somehow with #146.